style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #f5f5f5;
+  font-family: Poppins, sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+#app {
+  background: #fff;
+  width: 100vw;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 50px;
+  padding: 20px;
+}
+
+.header h2 {
+  color: #6070ff;
+}
+
+.showcase {
+  background-image: url("./img/header-shapes-mobile.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 75vh;
+  padding: 20px;
+}
+
+.showcase h3 {
+  padding-top: 45px;
+  font-size: 30px;
+  width: 200px;
+  font-weight: 700;
+  margin-bottom: 10px;
+}
```

Comment:
- [ ] To better follow the original Figma design kindly use the design given font-size and line-height
![image](https://user-images.githubusercontent.com/73140461/148434653-37c68f56-f57b-452f-8821-3c53d7564ef0.png)